clients/lightrider-entry.js

```diff
@@ -49,7 +49,7 @@ async function runLighthouseInLR(connection, url, flags, {lrDevice, categoryIDs,
     }
 
     // pre process the LHR for proto
-    if (flags.output === 'json' && typeof results.report === 'string') {
+    if (flags.output === ['json'] && typeof results.report === 'string') {
```

Comment:
unfortunately this doesn't work, the equality operator here is checking references not value.

```
> const array = ['json'];
< undefined
> ['json'] === array
< false
> ['json'] == array
< false
```

will need to be more explicit
```
flags.output && flags.output.length === 1 && flags.output[0] === 'json'
```

Reply:
#saving me from myself